### ISSUE TITLE
[Question] hardware used and training time

### ISSUE THREAD

**Issue by QuanticDisaster** [POSTECH-CVLab/FastPointTransformer#1]
Hi

The paper mentions an inference time extremely reduced compared to the original PointTransformer, but I was also curious about the time it took to train the model. What GPUs did you use and how much time did it take to train compared to PointTransformer ?

Thx a lot

**Comment by chrockey** [comment]
Hi!

Thanks for your interest in our work 😄 .
We train Fast Point Transformer on S3DIS with a single A100-80GB GPU (total 80GB) for 30k iterations with batch size 8, and it took 20 hours.
For Point Transformer, we train the model with four RTX 3090 GPUs (total 96GB) for ~40k iterations with batch size 16, and it took ~16 hours.
For latency, we measure the inference time of both Fast Point Transformer and Point Transformer on the same GPU, RTX 3090.

FYI, the reason why we did not use multi-GPU training is that we found a weird performance drop when we use multi-GPU DDP training with PyTorch Lightning and MinkowskiEngine.
This weird problem happened with both MinkowskiNet and Fast Point Transformer.
Therefore, we decided not to use it for our work.
